templates/how-to.md

```diff
@@ -0,0 +1,43 @@
+# Title Here
+
+#### Table of Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Step 1](#step-1)
+- [Step 2](#step-2)
+- [Step 3](#step-3)
+- [Step 4](#step-4)
+- [Step 5](#step-5)
+- [Next Steps](#next-steps)
+
+
+### Introduction
```

Comment:
Why are the introduction, prereqs, and next steps level 3 headers, whereas steps are level 2? (Not a rhetorical question, just curious if it's intentional.)

Reply:
SEO purposes - telling search engine the step titles are more relevant than the words "Intro" and "prereqs" - this is inline with other documentation sites such as digital ocean👍

Reply:
Oh, perfect!